docs/tasks/task-ods-start.adoc

```diff
@@ -0,0 +1,108 @@
+// Document generated by internal/documentation/tasks.go from template.adoc.tmpl; DO NOT EDIT.
+
+= ods-start-v0-1-0
+
+The task will clone a repo from the provided url into the workspace.
+
+== Parameters
+
+[cols="1,1,2"]
+|===
+| Parameter | Default | Description
+
+
+| image
+| image-registry.openshift-image-registry.svc:5000/ods/ods-start:latest
+| The location of the image.
+
+
+| url
+| 
+| git url to clone
+
+
+| git-full-ref
+| 
+| git revision to checkout (branch, tag, sha, ref…)
+
+
+| refspec
+| 
+| (optional) git refspec to fetch before checking out revision
+
+
+| submodules
+| true
+| defines if the resource should initialize and fetch the submodules
+
+
+| depth
+| 1
+| performs a shallow clone where only the most recent commit(s) will be fetched
+
+
+| ssl-verify
+| true
+| defines if http.sslVerify should be set to true or false in the global git config
+
+
+| delete-existing
+| true
+| clean out the contents of the repo's destination directory (if it already exists) before trying to clone the repo there
+
+
+| http-proxy
+| 
+| git HTTP proxy server for non-SSL requests
+
+
+| https-proxy
+| 
+| git HTTPS proxy server for SSL requests
+
+
+| no-proxy
+| 
+| git no proxy - opt out of proxying HTTP/HTTPS requests
+
+
+| verbose
+| true
+| log the commands used during execution
+
+
+| project
+| 
+| project
+
+
+| component
+| 
+| component
```

Comment:
Would be great to elaborate a bit more on the project, component, repository, pr-key and, pr-base description